bedrock/firefox/firefox_details.py

```diff
@@ -157,28 +167,25 @@ def get_download_url(self, platform, language, version, product='firefox'):
             language = 'ja-JP-mac'
 
         if version == self.latest_version('aurora'):
-            return self._get_aurora_download_url(platform, language, version)
+            # Download links are different for localized versions
+            if language == 'en-US':
+                if platform == 'Windows':
```

Comment:
People are still working on it: [Bug 797208](https://bugzilla.mozilla.org/show_bug.cgi?id=797208)